canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss

```diff
@@ -21,6 +21,15 @@
 	.bx--form-item {
 		margin-bottom: $spacing-02;
 	}
+	.tooltip-container{
+		height: auto;
+		display: flex;
+		align-items: center;
+		cursor: pointer;
+	}
+	&[data-id="properties-radioInteger"] .properties-radioset-panel {
```

Comment:
This doesn't work. This css would only work for the test harness.  Why do we need special css for numbers?  Shouldn't seem to matter if it's a string or integer with how css is handled.